notebooks/HASP/WavelengthAdjustment/WavelengthAdjustment.ipynb

```diff
@@ -0,0 +1,2632 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #5.    x1ds = glob.glob(f"{cos_unshifted}/*x1d*")
</pre><p>Lets make all this consistent and use pathlib.glob instead:</p>

<pre class="ql-syntax" spellcheck="false">x1ds = list(cos_unshifted.glob("*x1d*"))
</pre><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/spacetelescope/hst_notebooks/pull/297/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='13'/>